src/components/trace/hat-script-graph.ts

```diff
@@ -323,43 +277,33 @@ class HatScriptGraph extends LitElement {
     graphStart = false
   ) {
     const trace: any = this.trace.trace[path];
-    const track_path = trace ? [0, 1] : [];
     const repeats = this.trace?.trace[`${path}/repeat/sequence/0`]?.length;
     return html`
-      <hat-graph
-        .track_start=${track_path}
-        .track_end=${track_path}
+      <hat-graph-branch
         tabindex=${trace === undefined ? "-1" : "0"}
-        branching
         @focus=${this.selectNode(node, path)}
-        class=${classMap({
-          track: path in this.trace.trace,
-          active: this.selected === path,
-        })}
+        ?track=${path in this.trace.trace}
+        ?active=${this.selected === path}
       >
         <hat-graph-node
           .graphStart=${graphStart}
           .iconPath=${mdiRefresh}
-          class=${classMap({
-            track: trace,
-          })}
+          ?track=${path in this.trace.trace}
           slot="head"
           nofocus
         ></hat-graph-node>
         <hat-graph-node
           .iconPath=${mdiArrowUp}
+          ?track=${repeats > 1}
           nofocus
-          class=${classMap({
-            track: track_path.includes(1),
-          })}
-          .badge=${repeats}
+          .badge=${repeats > 1 ? repeats : undefined}
         ></hat-graph-node>
-        <hat-graph>
+        <div ?track=${trace}>
```

Comment:
What's the purpose of this property for a div?

Reply:
It is used in the branch to draw a line to it (I think, will check)

Reply:
You are right. All good then.